### ISSUE TITLE
Multiple consecutive line breaks in Textarea don't work under Processing3.0b4

### ISSUE THREAD

**Issue by jwmatthys** [sojamo/controlp5#20]
The following code worked fine under 3.0b3, but with 3.0b4 everything is truncated after the '\n':

```java
ControlP5 cp5;
Textarea myTextarea;

void setup() {
  size(200, 200);
  background(0);
  cp5 = new ControlP5(this);

  myTextarea = cp5.addTextarea("txt")
    ;
  myTextarea.setText("Line1\n\nLine3");
}
```
![screenshot from 2015-08-26 13 48 27](https://cloud.githubusercontent.com/assets/716646/9502851/3df0b98c-4bf9-11e5-9645-3c5a2ce8f2aa.png)

ControlP5 2.2.5, tested on OSX, Windows7, and Ubuntu Linux (64 and 32 bit).

**Comment by sojamo** [comment]
Indeed multi line breaks do not work properly with Processing 3.0b5 anymore. ControlP5's Textarea is using processing's text() method to render text which then also takes care of line breaks. the attached Processing example shows the same behaviour, after a double line break the subsequent text is not displayed. A single line break behaves as expected though. Since Processing 3 is currently in beta I will monitor this behaviour but will not take action in finding a workaround to this misbehaviour. maybe you want to open an issue with the processing project?

```java
void setup() {
  size(300,300);
}

void draw() {
  background(0);
  String s = "Hello\n\nWorld.";
  fill(255);
  text(s,0,0,100,400);
}
```

**Comment by jwmatthys** [comment]
Thanks. I opened an issue here: https://github.com/processing/processing/issues/3736

**Comment by sojamo** [comment]
ok, closing this, seem to be a processing issue.

**Comment by jwmatthys** [comment]
Problem resolved with processing 3.0b6.
